src/Screens/Logs/index.tsx

```diff
@@ -34,22 +34,20 @@ interface LogsProps {
 const Logs: FunctionComponent<LogsProps> = ({ navigation }) => {
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
-  const diveLogs = Object.values(useAppSelector(selectAllDiveLogs));
+  const diveLogs = useAppSelector(selectOrderedDiveLogs);
   const diveLogsIsLoading = useAppSelector(selectDiveLogsLoadingState);
   const authToken = useAppSelector(selectAuthToken);
   const user = useAppSelector(selectUser);
 
   useEffect(() => {
-    navigation.addListener('focus', () => {
```

Comment:
this was causing this function to get queued up a bunch and then sometimes make 5-10 API calls, and if you switched accounts it would also fetch the old account and there would be a flash of rendering

Reply:
The intention was to refetch the dive logs after creating a new log (in this case every time the screen is in focus). I think we can remove it and I'll find a way to trigger fetching after creating a dive log.

Reply:
it appears to me that just having navigation in the array that triggers the effect works well enough to do that, but i'm not super familiar with this. the other option i think would be to only have the effect run once one page load, but leave the navigation listener

Reply:
I can make a call to refresh the logs in state post submit. If you can create a ticket and add me to that, I'll do it today.